packages/playground/src/calculator/pricing_calculator.vue

```diff
@@ -0,0 +1,295 @@
+                >
+                  <VSwitch color="primary" inset label="With a Public IPv4" hide-details v-model="resources.ipv4" />
+                </InputTooltip>
+              </VCol>
+
+              <VCol
+                cols="4"
+                v-if="userBalance"
+                @vue:unmounted="
+                  resources.useCurrentBalance = true;
+                  priceTask.run();
+                "
+              >
+                <input-tooltip inline tooltip="Use current balance to calculate the discount.">
+                  <VSwitch
+                    color="primary"
+                    inset
+                    label="Use current balance"
+                    hide-details
+                    v-model="resources.useCurrentBalance"
+                  />
+                </input-tooltip>
+              </VCol>
+            </VRow>
+          </VForm>
+
+          <VDivider class="my-8" />
+          <VRow v-if="valid && priceTask.error">
+            <VCol cols="12">
+              <VAlert type="error" :text="normalizeError(priceTask.error, 'Failed to calculate price.')" />
+            </VCol>
+          </VRow>
+
+          <VRow class="text-center text-body-1 text-black" v-else-if="valid">
+            <VCol cols="6">
+              <div
+                class="rounded pa-4 h-100 d-flex justify-center align-center"
+                :style="{ background: computePackageColor(priceTask.data?.dedicatedPackage.package) }"
+              >
+                <p>
+                  Cost of reserving a <strong v-text="'Dedicated Node'" /> of the same specifications<br />
+                  <strong v-text="'Loading...'" v-if="priceTask.loading" />
+                  <strong v-else v-text="dedicatedPriceUSD + ' USD/month, ' + dedicatedPriceTFT + ' TFT/month'" />. A
+                  user can reserve an entire node then use it exclusively to deploy solutions
+                </p>
+              </div>
+            </VCol>
+            <VCol cols="6">
+              <div
+                class="rounded pa-4 h-100 d-flex justify-center align-center"
+                :style="{ background: computePackageColor(priceTask.data?.sharedPackage.package) }"
+              >
+                <p>
+                  Cost of reservation on a <strong v-text="'Shared Node'" /><br />
+                  <strong v-text="'Loading...'" v-if="priceTask.loading" />
+                  <strong v-else v-text="sharedPriceUSD + ' USD/month, ' + sharedPriceTFT + ' TFT/month'" />. Shared
+                  Nodes allow several users to host various workloads on a single node
+                </p>
+              </div>
+            </VCol>
+          </VRow>
+
+          <VRow v-else>
+            <VCol cols="12">
+              <VAlert type="error">
+                Please provide a valid data inorder to calculate your deployment price correctly.
```

Comment:
I think the alert should be more clear, it should mention which field is invalid. what do you think?

Reply:
there is already an error shown on that specific field ... we can know instantly which field has error without checking every field over and over